conf/radiusd/tls.conf.defaults

```diff
@@ -4,7 +4,8 @@ dh_file = ${certdir}/dh
 ca_path = ${cadir}
 cipher_list = DEFAULT
 ecdh_curve = prime256v1
-disable_tlsv1_2 = yes
+tls_min_version = 1.0
```

Comment:
I suggest to stay with TLS 1.2 as `tls_min_version` like you proposed on #5808. Users who want to keep TLS 1.0 need to changed it here.